pymc3/distributions/continuous.py

```diff
@@ -1654,6 +1655,93 @@ def logcdf(self, value):
         )
 
 
+class AsymmetricLaplace(Continuous):
+    r"""
+    Asymmetric-Laplace log-likelihood.
+
+    The pdf of this distribution is
+
+    ..math::
```

Comment:
I do not know much about sphinx directives, but let's sync with other PyMC3 distributions.
```suggestion
    .. math::
```